asreview/webapp/tests/test_api/test_remote_auth.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from jinja2.exceptions import TemplateNotFound
+from asreview.webapp.authentication.models import User
+from asreview.webapp.tests.utils.misc import custom_remote_auth_headers
+
+# ###################
+# NO LOGIN WHEN HEADER NOT SET
+# ###################
+
+
+@pytest.mark.parametrize("uri", ["/", "/signup"])
+def test_no_login_without_header(client_remote_auth, uri):
+    r = client_remote_auth.get(uri)
+    assert r.status_code == 302
+    assert r.location == "/signin"
+
+
+# ###################
+# LOGIN WHEN HEADER SET
+# ###################
+
+
+@pytest.mark.parametrize("uri", ["/", "/signup"])
+def test_login_with_header(client_remote_auth, uri):
+    user_identifier = "foo"
+    custom_headers = custom_remote_auth_headers(identifier=user_identifier)
+
+    def get_uri(path):
+        return client_remote_auth.get(uri, follow_redirects=True, **custom_headers)
+
+    def find_user(identifier):
+        return User.query.filter(User.identifier == identifier).one_or_none()
+
+    user = find_user(user_identifier)
+    assert user is None
+
+    with pytest.raises(TemplateNotFound):
```

Comment:
Hitting `/` or `/signup` while testing locally throws a template error, presumably because the templates are not setup in this context. But in the `compile_and_test` workflow ([here](https://github.com/asreview/asreview/actions/runs/12072095964/job/33777599102)) the templates are loaded, so this test fails. Any advice on how to handle this?

Reply:
I'm looking into this. I don't see what's going on at the moment.

Reply:
Any luck checking in to this?